src/kernels/MIOpenBatchNormBwdSpatial.cl

```diff
@@ -293,7 +293,7 @@ MIOpenBatchNormBwdSpatial(const __global _FLOAT* __restrict x_in,
 #if(MIO_BN_USESAVED == 0)
     //==== CALC MEAN and VARIANCE ONCE AGAIN =======================
     _FLOAT_PREC variance = (_FLOAT_PREC)0.;
-#if(MIO_BN_HW >= 4096)
+#if(0)
```

Comment:
@muralinr  Please follow the workaround protocol to document this effort and ensure that we are able to re-enable it again in the future with a proper fix.

Reply:
[Here is a link](https://github.com/ROCmSoftwarePlatform/MIOpen/issues/561)

Reply:
@muralinr Are you going to quickly make necessary changes? If not, then I will do.

Reply:
Hi Artem, I am coming up with proper fix without workaround.  I pushed changes. I am working with framework teams to test these changes..